From 0f3065a9a790a27a264aea1f8c0aad6341b18192 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cl=C3=A9ment=20Le=20Goffic?= <clement.legoffic@foss.st.com>
Date: Thu, 2 May 2024 13:49:03 +0200
Subject: [PATCH 0859/1695] dts: stm32: add console to stm32mp215f-dk
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Enable usart2 on stm32mp215f-dk.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 729223eaf0551ec095e143d4f269b6dd206d2336 dts: stm32: add console to stm32mp215f-dk]
Change-Id: I6ada936b11ee1566b8345bc453bf3e73c0ce417e
Signed-off-by: Clément Le Goffic <clement.legoffic@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/377758
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp215f-dk.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp215f-dk.dts b/core/arch/arm/dts/stm32mp215f-dk.dts
index 1ff8945d2..265971c15 100644
--- a/core/arch/arm/dts/stm32mp215f-dk.dts
+++ b/core/arch/arm/dts/stm32mp215f-dk.dts
@@ -79,3 +79,10 @@
 &saes {
 	status = "okay";
 };
+
+&usart2 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&usart2_pins_a>;
+	status = "okay";
+};
+
-- 
2.47.2

